cmd/debug.go

```diff
@@ -20,8 +20,13 @@ func newDebugCmd(rootConfig *RootCommandConfig) *cobra.Command {
 	// debug Cmd represents the debug command
 	var debugCmd = &cobra.Command{
 		Use:   "debug",
-		Short: "Run the local Appsody environment in debug mode",
-		Long:  `This starts a docker based continuous build environment for your project with debugging enabled.`,
+		Short: "Debug your Appsody project",
+		Long:  `Start a container-based continuous build environment for your Appsody project, with debugging enabled.`,
+		Example: `  appsody debug --docker-options "--privileged"
+  Starts debugging environment and passes the "--privileged" option to the docker run as a flag.
+  
+  appsody debug --name my-project-dev2 -p 3001:3000
+  Starts debugging environment, names development container "my-project-dev2" and binds container port 3000 to host port 3001.`,
```

Comment:
- Articles again :)  "The debugging environment", "the development container", "the container port". 
- Comma before the "and" at the end of a list, ie "Starts xxx, names xxx, and binds xxx".
- Do we need to say a bit more about the development container, or will it be obvious to the user? eg "Starts the debugging environment with a development container named ...". As a non-expert, I'm wondering where the development container comes from?

Reply:
I think it should be obvious enough to the user? The `appsody <run|test|debug>` commands all run a `docker run` command which starts the development container

Reply:
Fair enough :)